src/ripple/overlay/impl/Cluster.cpp

```diff
@@ -97,12 +98,13 @@ std::unique_ptr<Cluster>
 make_Cluster (Config const& config, beast::Journal j)
 {
     static boost::regex const re (
-        "^"                         // start of line
-        "(?:\\s*)"                  // whitespace (optional)
-        "([a-zA-Z0-9]*)"            // Node identity
-        "(?:\\s*)"                  // whitespace (optional)
-        "(.*\\S*)"                  // <value>
-        "(?:\\s*)"                  // whitespace (optional)
+        "[[:space:]]*"            // skip leading whitespace
+        "([[:alnum:]]+)"          // node identity
+        "(?:"                     // begin optional comment block
+        "[[:space:]]+"            // (skip any amount of whitespace)
+        "(.*[^[:space:]]+)"       // the comment (at least non-whitespace)
+        "[[:space:]]*"            // (skip any amount of whitespace)
+        ")?"                      // end optional comment block
```

Comment:
This repeats the regex in ValidatorList. We should put the parsing of the key/comment from a string into one function.